service/handlers.go

```diff
@@ -1133,6 +1137,19 @@ func (h handler) createToken(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	tokens, err := h.dbClient.ListTokenEntries(ctx, projectName)
+	if err != nil {
+		level.Error(l).Log("message", "error listing tokens from DB", "error", err)
+		h.errorResponse(w, "error listing tokens", http.StatusInternalServerError)
+		return
+	}
+
+	if len(tokens) == numOfTokensLimit {
```

Comment:
nit: should we make this `>=` in case somebody messes with the db/in case 2 requests go off at the same time or something

Reply:
alternatively we go down the rabbit hole of using some mutex/locking to make sure this actually never gets exceeded but that seems overkill for this

Reply:
Ah, good call out. I'll make that change.

I did think about using a mutex, but I, too, also thought it was overkill, at least for now.